client/tests/cypress/integration/components/HomePage.spec.jsx

```diff
@@ -0,0 +1,15 @@
+describe('Home Page', () => {
+  it('should find the text ANDELA and a sign in button on the home page', () => {
+    // Should visit the application landing page
+    cy.visit('localhost:8000');
```

Comment:
We can consider fetching this type of urls from an env variable. See the comment below for more details. If you are unable to access the env variable at this point, ignore this

Reply:
I added a cypress.env.json file that sets environment variables used for testing.